### DISCUSSION TITLE
使用无线推送的时候，目标文件名还是会被转化成拼音文件名

### DISCUSSION THREAD

**Discussion by allanmeng** [Cirn09/calibre-do-not-translate-my-path#21] [General]
在使用Calibre的无线连接，向Kindle的KOReader推送文件的时候，目标文件名还是会被转化成拼音。 这个问题是否也是该项目的目标？谢谢。 
![微信图片编辑_20240123021925](https://github.com/Cirn09/calibre-do-not-translate-my-path/assets/11768374/8cd1c030-5650-4a2c-abda-54787e3be333)

**Comment by Cirn09** [comment]
https://github.com/Cirn09/calibre-do-not-translate-my-path/issues/6

**Reply by allanmeng** [reply]
收到，谢谢！

**Comment by Cirn09** [comment]
可以了，快试试基于插件的 v3 版本吧
https://github.com/Cirn09/calibre-do-not-translate-my-path/releases/tag/v3.0.0
